doc/classes/Curve3D.xml

```diff
@@ -138,7 +136,7 @@
 			<param index="1" name="cubic" type="bool" default="false" />
 			<param index="2" name="apply_tilt" type="bool" default="false" />
 			<description>
-				Similar with [code]interpolate_baked()[/code]. The return value is [code]Transform3D[/code], with [code]origin[/code] as point position, [code]basis.x[/code] as sideway vector, [code]basis.y[/code] as up vector, [code]basis.z[/code] as forward vector. When the curve length is 0, there is no reasonable way to calculate the rotation, all vectors aligned with global space axes.
+				Returns a [Transform3D] with [code]origin[/code] as point position, [code]basis.x[/code] as sideway vector, [code]basis.y[/code] as up vector, [code]basis.z[/code] as forward vector. When the curve length is 0, there is no reasonable way to calculate the rotation, all vectors aligned with global space axes. See also [method sample_baked].
```

Comment:
This method was renamed long ago, but this was missed. I should try to add checks for these as well.

Reply:
Well, I tried, and caught 118 more potential issues. Some may be bad captures, as I cannot rule out a possibility of making mistakes in code or not accounting for something, but yeah... I won't be adding it to this PR just yet, can do it as a follow-up.

<details>
```

WARNING: @GDScript.xml: Found a string "@icon" that matches the @GDScript.@icon annotation in annotation "@icon" description.
WARNING: @GDScript.xml: Found a string "@icon" that matches the @GDScript.@icon annotation in annotation "@icon" description.
WARNING: @GDScript.xml: Found a string "@tool" that matches the @GDScript.@tool annotation in annotation "@tool" description.
WARNING: @GlobalScope.xml: Found a string "cubic_interpolate" that matches the @GlobalScope.cubic_interpolate method in method "cubic_interpolate_angle_in_time" description.
WARNING: @GlobalScope.xml: Found a string "Vector2.ZERO" that matches the Vector2.ZERO constant in method "type_convert" description.
WARNING: @GlobalScope.xml: Found a string "wrapf" that matches the @GlobalScope.wrapf method in method "wrapf" description.
WARNING: AStar3D.xml: Found a string "_compute_cost" that matches the AStar3D._compute_cost method in class "AStar3D" description.
WARNING: AStar3D.xml: Found a string "reserve_space" that matches the AStar3D.reserve_space method in method "get_point_capacity" description.
WARNING: AudioStreamPlayer.xml: Found a string "stream_paused" that matches the AudioStreamPlayer.stream_paused member in property "stream_paused" description.
WARNING: AudioStreamPlayer2D.xml: Found a string "stream_paused" that matches the AudioStreamPlayer2D.stream_paused member in property "stream_paused" description.
WARNING: BaseMaterial3D.xml: Found a string "distance_fade_max_distance" that matches the BaseMaterial3D.distance_fade_max_distance member in property "distance_fade_max_distance" description.
WARNING: BaseMaterial3D.xml: Found a string "distance_fade_min_distance" that matches the BaseMaterial3D.distance_fade_min_distance member in property "distance_fade_max_distance" description.
WARNING: BaseMaterial3D.xml: Found a string "distance_fade_max_distance" that matches the BaseMaterial3D.distance_fade_max_distance member in property "distance_fade_max_distance" description.
WARNING: BaseMaterial3D.xml: Found a string "distance_fade_min_distance" that matches the BaseMaterial3D.distance_fade_min_distance member in property "distance_fade_max_distance" description.
WARNING: BaseMaterial3D.xml: Found a string "distance_fade_min_distance" that matches the BaseMaterial3D.distance_fade_min_distance member in property "distance_fade_min_distance" description.
WARNING: BaseMaterial3D.xml: Found a string "distance_fade_max_distance" that matches the BaseMaterial3D.distance_fade_max_distance member in property "distance_fade_min_distance" description.
WARNING: BaseMaterial3D.xml: Found a string "distance_fade_max_distance" that matches the BaseMaterial3D.distance_fade_max_distance member in property "distance_fade_min_distance" description.
WARNING: BaseMaterial3D.xml: Found a string "distance_fade_min_distance" that matches the BaseMaterial3D.distance_fade_min_distance member in property "distance_fade_min_distance" description.
WARNING: Basis.xml: Found a string "Basis" that matches one of the known classes in constant "IDENTITY" description.
WARNING: Basis.xml: Found a string "Basis" that matches the Basis.Basis constructor in constant "IDENTITY" description.
WARNING: Button.xml: Found a string "focus" that matches the Button.focus theme item in theme item "focus" description.
WARNING: Camera3D.xml: Found a string "fov" that matches the Camera3D.fov member in property "fov" description.
WARNING: CharacterBody2D.xml: Found a string "Vector2.UP" that matches the Vector2.UP constant in property "up_direction" description.
WARNING: CharacterBody2D.xml: Found a string "Vector2.UP" that matches the Vector2.UP constant in method "get_floor_angle" description.
WARNING: CharacterBody3D.xml: Found a string "Vector3.UP" that matches the Vector3.UP constant in property "up_direction" description.
WARNING: CharacterBody3D.xml: Found a string "Vector3.UP" that matches the Vector3.UP constant in method "get_floor_angle" description.
WARNING: CodeEdit.xml: Found a string "indent_use_spaces" that matches the CodeEdit.indent_use_spaces member in property "indent_size" description.
WARNING: CollisionObject2D.xml: Found a string "input_event" that matches the CollisionObject2D.input_event signal in method "_input_event" description.
WARNING: CollisionPolygon3D.xml: Found a string "polygon" that matches the CollisionPolygon3D.polygon member in property "polygon" description.
WARNING: DampedSpringJoint2D.xml: Found a string "damping" that matches the DampedSpringJoint2D.damping member in property "damping" description.
WARNING: DisplayServer.xml: Found a string "FEATURE_NATIVE_DIALOG" that matches the DisplayServer.FEATURE_NATIVE_DIALOG enum value in method "file_dialog_show" description.
WARNING: EditorExportPlatformMacOS.xml: Found a string "privacy/address_book_usage_description" that matches the EditorExportPlatformMacOS.privacy/address_book_usage_description member in property "codesign/entitlements/address_book" description.
WARNING: EditorExportPlatformMacOS.xml: Found a string "privacy/microphone_usage_description" that matches the EditorExportPlatformMacOS.privacy/microphone_usage_description member in property "codesign/entitlements/audio_input" description.
WARNING: EditorExportPlatformMacOS.xml: Found a string "privacy/calendar_usage_description" that matches the EditorExportPlatformMacOS.privacy/calendar_usage_description member in property "codesign/entitlements/calendars" description.
WARNING: EditorExportPlatformMacOS.xml: Found a string "privacy/camera_usage_description" that matches the EditorExportPlatformMacOS.privacy/camera_usage_description member in property "codesign/entitlements/camera" description.
WARNING: EditorExportPlatformMacOS.xml: Found a string "privacy/location_usage_description" that matches the EditorExportPlatformMacOS.privacy/location_usage_description member in property "codesign/entitlements/location" description.
WARNING: EditorExportPlatformMacOS.xml: Found a string "privacy/photos_library_usage_description" that matches the EditorExportPlatformMacOS.privacy/photos_library_usage_description member in property "codesign/entitlements/photos_library" description.
WARNING: EditorExportPlugin.xml: Found a string "add_ios_framework" that matches the EditorExportPlugin.add_ios_framework method in method "add_ios_embedded_framework" description.
WARNING: Environment.xml: Found a string "adjustment_enabled" that matches the Environment.adjustment_enabled member in property "adjustment_brightness" description.
WARNING: Environment.xml: Found a string "adjustment_enabled" that matches the Environment.adjustment_enabled member in property "adjustment_color_correction" description.
WARNING: Environment.xml: Found a string "adjustment_enabled" that matches the Environment.adjustment_enabled member in property "adjustment_contrast" description.
WARNING: Environment.xml: Found a string "adjustment_enabled" that matches the Environment.adjustment_enabled member in property "adjustment_saturation" description.
WARNING: GPUParticles3D.xml: Found a string "process_material" that matches the GPUParticles3D.process_material member in class "GPUParticles3D" description.
WARNING: GPUParticles3D.xml: Found a string "amount" that matches the GPUParticles3D.amount member in property "one_shot" description.
WARNING: HTTPRequest.xml: Found a string "request_completed" that matches the HTTPRequest.request_completed signal in property "accept_gzip" description.
WARNING: HTTPRequest.xml: Found a string "accept_gzip" that matches the HTTPRequest.accept_gzip member in property "accept_gzip" description.
WARNING: HTTPRequest.xml: Found a string "request_completed" that matches the HTTPRequest.request_completed signal in property "accept_gzip" description.
WARNING: Image.xml: Found a string "data" that matches the Image.data member in method "get_mipmap_offset" description.
WARNING: JavaScriptBridge.xml: Found a string "eval" that matches the JavaScriptBridge.eval method in method "eval" description.
WARNING: Joint2D.xml: Found a string "bias" that matches the Joint2D.bias member in property "bias" description.
WARNING: Joint2D.xml: Found a string "bias" that matches the Joint2D.bias member in property "bias" description.
WARNING: Label.xml: Found a string "lines_skipped" that matches the Label.lines_skipped member in property "lines_skipped" description.
WARNING: Line2D.xml: Found a string "texture_mode" that matches the Line2D.texture_mode member in property "texture" description.
WARNING: LinkButton.xml: Found a string "focus" that matches the LinkButton.focus theme item in theme item "focus" description.
WARNING: MarginContainer.xml: Found a string "margin_bottom" that matches the MarginContainer.margin_bottom theme item in theme item "margin_bottom" description.
WARNING: MarginContainer.xml: Found a string "margin_left" that matches the MarginContainer.margin_left theme item in theme item "margin_left" description.
WARNING: MarginContainer.xml: Found a string "margin_right" that matches the MarginContainer.margin_right theme item in theme item "margin_right" description.
WARNING: MarginContainer.xml: Found a string "margin_top" that matches the MarginContainer.margin_top theme item in theme item "margin_top" description.
WARNING: MenuButton.xml: Found a string "switch_on_hover" that matches the MenuButton.switch_on_hover member in property "switch_on_hover" description.
WARNING: NavigationPathQueryParameters2D.xml: Found a string "map" that matches the NavigationPathQueryParameters2D.map member in property "map" description.
WARNING: NavigationPathQueryParameters3D.xml: Found a string "map" that matches the NavigationPathQueryParameters3D.map member in property "map" description.
WARNING: NavigationRegion2D.xml: Found a string "enter_cost" that matches the NavigationRegion2D.enter_cost member in property "enter_cost" description.
WARNING: NavigationRegion2D.xml: Found a string "travel_cost" that matches the NavigationRegion2D.travel_cost member in property "travel_cost" description.
WARNING: NavigationRegion3D.xml: Found a string "enter_cost" that matches the NavigationRegion3D.enter_cost member in property "enter_cost" description.
WARNING: NavigationRegion3D.xml: Found a string "travel_cost" that matches the NavigationRegion3D.travel_cost member in property "travel_cost" description.
WARNING: Node.xml: Found a string "_ready" that matches the Node._ready method in method "_ready" description.
WARNING: Node.xml: Found a string "_ready" that matches the Node._ready method in method "request_ready" description.
WARNING: Node.xml: Found a string "_ready" that matches the Node._ready method in method "request_ready" description.
WARNING: Node.xml: Found a string "_ready" that matches the Node._ready method in method "request_ready" description.
WARNING: Node.xml: Found a string "_ready" that matches the Node._ready method in method "request_ready" description.
WARNING: Object.xml: Found a string "Object.connect" that matches the Object.connect method in method "connect" description.
WARNING: Object.xml: Found a string "Signal.connect" that matches the Signal.connect method in method "connect" description.
WARNING: PackedByteArray.xml: Found a string "decompress" that matches the PackedByteArray.decompress method in method "decompress_dynamic" description.
WARNING: PackedByteArray.xml: Found a string "decompress" that matches the PackedByteArray.decompress method in method "decompress_dynamic" description.
WARNING: PackedScene.xml: Found a string "pack" that matches the PackedScene.pack method in class "PackedScene" description.
WARNING: PacketPeer.xml: Found a string "encode_buffer_max_size" that matches the PacketPeer.encode_buffer_max_size member in property "encode_buffer_max_size" description.
WARNING: ParticleProcessMaterial.xml: Found a string "emission_shape" that matches the ParticleProcessMaterial.emission_shape member in property "emission_box_extents" description.
WARNING: ParticleProcessMaterial.xml: Found a string "emission_shape" that matches the ParticleProcessMaterial.emission_shape member in property "emission_point_count" description.
WARNING: ParticleProcessMaterial.xml: Found a string "emission_shape" that matches the ParticleProcessMaterial.emission_shape member in property "emission_sphere_radius" description.
WARNING: PhysicsServer2D.xml: Found a string "RID" that matches one of the known classes in method "area_get_space" description.
WARNING: PhysicsServer2D.xml: Found a string "RID" that matches one of the known classes in method "body_get_space" description.
WARNING: PhysicsServer2D.xml: Found a string "RID" that matches one of the known classes in method "joint_make_pin" description.
WARNING: PhysicsServer3D.xml: Found a string "body_set_force_integration_callback" that matches the PhysicsServer3D.body_set_force_integration_callback method in method "body_set_force_integration_callback" description.
WARNING: Plane.xml: Found a string "d" that matches the Plane.d member in property "d" description.
WARNING: Plane.xml: Found a string "d" that matches the Plane.d member in property "normal" description.
WARNING: Polygon2D.xml: Found a string "texture" that matches the Polygon2D.texture member in property "color" description.
WARNING: Polygon2D.xml: Found a string "vertex_colors" that matches the Polygon2D.vertex_colors member in property "color" description.
WARNING: Polygon2D.xml: Found a string "uv" that matches the Polygon2D.uv member in property "texture" description.
WARNING: Polygon2D.xml: Found a string "texture" that matches the Polygon2D.texture member in property "texture_offset" description.
WARNING: Polygon2D.xml: Found a string "uv" that matches the Polygon2D.uv member in property "texture_scale" description.
WARNING: Polygon2D.xml: Found a string "texture" that matches the Polygon2D.texture member in property "texture_scale" description.
WARNING: Polygon2D.xml: Found a string "uv" that matches the Polygon2D.uv member in property "uv" description.
WARNING: Polygon2D.xml: Found a string "color" that matches the Polygon2D.color member in property "vertex_colors" description.
WARNING: ProjectSettings.xml: Found a string "xr/openxr/foveation_level" that matches the ProjectSettings.xr/openxr/foveation_level member in property "xr/openxr/foveation_dynamic" description.
WARNING: Quaternion.xml: Found a string "spherical_cubic_interpolate" that matches the Quaternion.spherical_cubic_interpolate method in method "spherical_cubic_interpolate_in_time" description.
WARNING: RichTextLabel.xml: Found a string "text" that matches the RichTextLabel.text member in property "text" description.
WARNING: RichTextLabel.xml: Found a string "focus" that matches the RichTextLabel.focus theme item in theme item "focus" description.
WARNING: RigidBody3D.xml: Found a string "Vector3.ZERO" that matches the Vector3.ZERO constant in property "inertia" description.
WARNING: Slider.xml: Found a string "grabber_area" that matches the Slider.grabber_area theme item in theme item "slider" description.
WARNING: SpinBox.xml: Found a string "custom_arrow_step" that matches the SpinBox.custom_arrow_step member in property "custom_arrow_step" description.
WARNING: SpinBox.xml: Found a string "prefix" that matches the SpinBox.prefix member in property "prefix" description.
WARNING: SpinBox.xml: Found a string "suffix" that matches the SpinBox.suffix member in property "suffix" description.
WARNING: TextEdit.xml: Found a string "focus" that matches the TextEdit.focus theme item in theme item "focus" description.
WARNING: TextureProgressBar.xml: Found a string "texture_over" that matches the TextureProgressBar.texture_over member in property "tint_over" description.
WARNING: TextureProgressBar.xml: Found a string "texture_progress" that matches the TextureProgressBar.texture_progress member in property "tint_progress" description.
WARNING: TextureProgressBar.xml: Found a string "texture_under" that matches the TextureProgressBar.texture_under member in property "tint_under" description.
WARNING: TileSet.xml: Found a string "add_source" that matches the TileSet.add_source method in method "get_next_source_id" description.
WARNING: TileSetAtlasSource.xml: Found a string "margins" that matches the TileSetAtlasSource.margins member in method "get_atlas_grid_size" description.
WARNING: TileSetAtlasSource.xml: Found a string "texture_region_size" that matches the TileSetAtlasSource.texture_region_size member in method "get_atlas_grid_size" description.
WARNING: Transform2D.xml: Found a string "is_equal_approx" that matches the Transform2D.is_equal_approx method in method "is_equal_approx" description.
WARNING: Transform3D.xml: Found a string "is_equal_approx" that matches the Transform3D.is_equal_approx method in method "is_equal_approx" description.
WARNING: Tree.xml: Found a string "item_selected" that matches the Tree.item_selected signal in signal "multi_selected" description.
WARNING: Tree.xml: Found a string "select_mode" that matches the Tree.select_mode member in signal "multi_selected" description.
WARNING: Vector2.xml: Found a string "cubic_interpolate" that matches the Vector2.cubic_interpolate method in method "cubic_interpolate_in_time" description.
WARNING: Vector3.xml: Found a string "cubic_interpolate" that matches the Vector3.cubic_interpolate method in method "cubic_interpolate_in_time" description.
WARNING: Vector4.xml: Found a string "cubic_interpolate" that matches the Vector4.cubic_interpolate method in method "cubic_interpolate_in_time" description.
WARNING: VisualShaderNodeSwitch.xml: Found a string "op_type" that matches the VisualShaderNodeSwitch.op_type member in class "VisualShaderNodeSwitch" description.
WARNING: WorldBoundaryShape2D.xml: Found a string "Vector2.UP" that matches the Vector2.UP constant in property "normal" description.

Generating the index file...

118 warnings were found in the class reference XML. Please check the messages above.
```
</details>

Reply:
Well, I just fixed all of them anyway.